objects/cash_register.rb

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+
+# purchase method takes a floating number and adds that to the total
+# total method returns how much is owed
+# pay method takes one floating number for how much is paid
+# should return how much change is given
+# create a CashRegister class
+class CashRegister
+  def initialize(total = 0.0)
+    @total = total
+  end
+
+  def purchase(floating_number)
+    @total = floating_number + @total
```

Comment:
Best practice:  `@total += floating_number`

Maybe instead of `floating_number` you could use `amount`?  I.e. "I made a purchase for this amount" vs "I made a purchase for floating number".